ITEM-10828 ITEM-10903 add docs for user secrets (#1076)
fusedio/fused-docs: docs/python-sdk/api-reference/fused-user-secrets.mdx

---
id: fused-user-secrets
title: "fused.user_secrets"
sidebar_label: "fused.user_secrets"
---

# user_secrets

> Access user-owned secrets that persist across all execution environments. User secrets are private to the authenticated user — they are not shared with your team. Requires a paid plan.

Reviewer: ```suggestion
Access secrets that are private to you. User secrets are private to the authenticated user — they are not shared with your team. Requires a paid plan.
```
